sdks/java/core/src/test/java/org/apache/beam/sdk/metrics/MetricsTest.java

```diff
@@ -277,7 +286,8 @@ public static class CommittedMetricTests extends SharedTestBase {
       UsesCounterMetrics.class,
       UsesDistributionMetrics.class,
       UsesGaugeMetrics.class,
-      UsesStringSetMetrics.class
+      UsesStringSetMetrics.class,
+      UsesBoundedTrieMetrics.class
```

Comment:
Does this mean we lose (ordinary) counter coverage for runners not yet supporting bounded trie metrics?